app/views/play/level/modal/HeroVictoryModal.coffee

```diff
@@ -212,10 +212,12 @@ module.exports = class HeroVictoryModal extends ModalView
         when 'game-dev-hoc' then '57ee6f5786cf4e1f00afca2c' # game grove
         when 'game-dev-hoc-2' then '57b71dce7a14ff35003a8f71' # palimpsest
         else '541c9a30c6362edfb0f34479' # kithgard gates for dungeon
+      amazonLastLevel = @level.get('original') is '541875da4c16460000ab990f' # true names - for amazon kids-to-work-day
       lastLevel = @level.get('original') is lastLevelOriginal
       enough = elapsed >= 20 * 60 * 1000 or lastLevel
       tooMuch = elapsed > 120 * 60 * 1000
-      showDone = (elapsed >= 30 * 60 * 1000 and not tooMuch) or lastLevel
+      #showDone = (elapsed >= 30 * 60 * 1000 and not tooMuch) or lastLevel 
```

Comment:
If this comment is intentional, please add a comment explaining.  Otherwise, let's remove it.

Reply:
I kept this as comment because we want to revert back to it after july 3, i will add a comment explaining that